README.md

```diff
@@ -12,13 +12,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
 ## Quick Start
 
-1. Install `vagrant-service-manager` plugin
+1. Install `vagrant-service-manager` plugin:
 
         vagrant plugin install vagrant-service-manager
 
-2. Get the [Vagrantfile](Vagrantfile) and start the ADB using `vagrant up`. [More](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) documentation on setting up ADB.
+2. Get the [Vagrantfile](Vagrantfile).
```

Comment:
This needs to be a reference to the ADB repo.  The Vagrantfile in this repo is for testing and should possibly be deleted or augmented.

I suggest:

2. Get a Vagrantfile for your box.  Users of the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle/) should download a [Vagrantfile from the repository.](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components)